dkms.8.in

```diff
@@ -651,6 +651,16 @@ This optional directive functions very similarly to
 except that it matches against the kernel architecture. For example, if you set
 it to ="i.86", your module would not be built for ia32e, x86_64, amd64, s390, etc.
 .TP
```

Comment:
Nit: Please add "Closes: https://github.com/dell/dkms/issues/219" in the commit message. Thus github will auto-close the report.